### ISSUE TITLE
Activiteit: waarde aanpassen

### ISSUE THREAD

**Issue by Socioneers** [kgish/discourse-annotator-plugin#74]

**Comment by kgish** [comment]
Sorry, but I cannot recall this at the top of my head. Please provide more details so that I understand what this is.

**Comment by Socioneers** [comment]
Show input value of annotations here.
<img width="257" alt="screen shot 2016-10-13 at 14 56 24" src="https://cloud.githubusercontent.com/assets/11832800/19349820/618876da-9155-11e6-9b9f-875608b05d2e.png">

**Comment by kgish** [comment]
This is tricky, is this a must for the deadline?

**Comment by Socioneers** [comment]
I think it is not the most important issue for now..

**Comment by kgish** [comment]
The activity value will now reflect the time when an annotation is created, updated or deleted as well as when an annotation is liked or commented on.

This is accomplished by subscribing to the add, update, and destroy events and then hitting the `bumped_at` topic field with the current time (unix format).

```
PUT '/t/:topic_id/change-timestamp 
data = { timestamp: timestamp }
```

In the file `apply-discourse-annotator.js.es6` we have:
```javascript
var updateTopicTimestamp = function(annotation, action) {
    // Convert now to unix time
    var timestamp = Math.floor( new Date().getTime() / 1000);
    var url_topic_id = `/t/${annotation.topic_id}/change-timestamp`, \
        json = {timestamp: timestamp };
    $.ajax({
        url: url_topic_id,
        type: 'PUT',
        data: json,
        success: function(data) {
            console.log('Success!');
        },
        error: function() {
            console.error('Oeps, something went terribly wrong!');
        }
    });
};
```

This functionality can be configured in the admin plugin dashboard with the `update_topic_timestamp` configuration parameter.

This is a comma-separated list of actions when to update topic timestamp: annotation is created, updated or deleted (add,update,delete) or has been liked or commented on (like,comment).

By default all actions are enabled:
```
add,update,delete,like,comment
```

But if you prefer that one or more of the actions do not cause an update, remove them from the string. For example,
```
add,update,delete
```
Would only cause timestamp updates to occur when annotations are created, update or deleted, but NOT when liked or commented on.

![screenshot from 2016-10-15 21-32-17](https://cloud.githubusercontent.com/assets/79195/19412854/e807ef2c-931e-11e6-93b5-c7c138f5a2c9.png)

Now you will notice that when any of the actions named above take place, the activity value is update appropriately.

**Comment by Socioneers** [comment]
actions are not update appropriately, when testing

**Comment by Socioneers** [comment]
actions are not update appropriately, when testing

**Comment by Socioneers** [comment]
actions are not update appropriately, when testing

**Comment by kgish** [comment]
That's because during the dinsdag sessie it was disabled:

![screenshot from 2016-10-23 10-43-31](https://cloud.githubusercontent.com/assets/79195/19628227/d3dcddba-990d-11e6-9a03-a8d9c9510660.png)

So I turned it back on:

![screenshot from 2016-10-23 10-43-58](https://cloud.githubusercontent.com/assets/79195/19628231/f027a02c-990d-11e6-961b-1054f92509d0.png)

Please try again.

**Comment by Socioneers** [comment]
Hi Kiffin, it seem to be not working in production, does this maybe have to do with the https connection?

**Comment by kgish** [comment]
Are any errors generated? Please hit Ctrl-Shift-I to open up the chrome devtools console, clear the content, try again and send me the output?

**Comment by Socioneers** [comment]
We see only that admins time stamps are registrated. Normal users not.

<img width="1247" alt="screen shot 2016-10-25 at 22 39 12" src="https://cloud.githubusercontent.com/assets/11832800/19703583/9f78499e-9b04-11e6-91bc-44bdbae4cdb1.png">

**Comment by kgish** [comment]
This is not good, works fine in my development environment, and am assuming that it was verfiied on the test environment. Will need to investigate later.

For the time being, I disabled this by clearing out the `annotator update topic timestamp` configuration parameter in the admin dashboard plugins:

Before:
![screenshot from 2016-10-26 10-47-10](https://cloud.githubusercontent.com/assets/79195/19737789/b418e72c-9b69-11e6-8e11-365f31d8825e.png)

After:
![screenshot from 2016-10-26 10-47-30](https://cloud.githubusercontent.com/assets/79195/19737846/f1a7e4c6-9b69-11e6-8d1b-00afd6db0189.png)

Move back to the 'In progress' column.

**Comment by Socioneers** [comment]
That you got my reaction that it works as a admin in production and not as a normal user?

Thanks

**Comment by kgish** [comment]
Yes I did, thanks for the reminder. But that doesn't make sense at all. Will have a look.

**Comment by Socioneers** [comment]
That is also the same on test only admins are registert.

**Comment by Socioneers** [comment]
Are this and #103 related? < merge?

**Comment by kgish** [comment]
This fix is a temporary workaround, and once the pull request is accepted I can finish it off with the proper fix #103.
